### ISSUE TITLE
BSO COX changes

### ISSUE THREAD

**Issue by adam55419** [oldschoolgg/oldschoolbot#730]
## 🚀 Feature Proposal
I'll keep this short as possible;

Twisted bow Should be more bonus points than 9, DHCB+buckler is 1 point off...
Scythe shouldn't be 5 points. if you had no melee weapon but you had a dragon arrow then you could get the same bonus as a scythe, also the fact that DHL + avernic is 4 points more makes no sense.

And now half of the items people have gained haven't even been coded like max capes/imbued/uncharged/enchanted. 

Basically if you want a scythe for cox you are going to be paying 3b+ as there's not many charged versions for a bonus that's the same as 1 dragon arrow that's basically free.

edit: Also at a certain amount of points its actually possible to get 2 purples (not sure if this was added in but no one has seen 2 purples in 1 trip yet)

**Comment by Lajnux** [comment]
Most items are now coded in, and the point system will stay until a full rework at a much later time when we make it more dynamic.
